shell/components/form/Members/MembershipEditor.vue

```diff
@@ -56,20 +56,13 @@ export default {
   },
 
   async fetch() {
-    const userHydration = [
-      this.schema ? this.$store.dispatch(`rancher/findAll`, { type: this.type, opt: { force: true } }) : [],
-      this.$store.dispatch('rancher/findAll', { type: NORMAN.PRINCIPAL }),
-      this.$store.dispatch(`management/findAll`, { type: MANAGEMENT.ROLE_TEMPLATE }),
-      this.$store.dispatch(`management/findAll`, { type: MANAGEMENT.USER })
-    ];
-    const [allBindings] = await Promise.all(userHydration);
-
-    const bindings = allBindings
-      .filter(b => normalizeId(get(b, this.parentKey)) === normalizeId(this.parentId));
-
-    this.$set(this, 'lastSavedBindings', [...bindings]);
+    this.$store.dispatch('rancher/findAll', { type: NORMAN.PRINCIPAL });
+    this.$store.dispatch(`management/findAll`, { type: MANAGEMENT.ROLE_TEMPLATE });
+    this.$store.dispatch(`management/findAll`, { type: MANAGEMENT.USER });
 
     // Add the current user as the project owner. This will get created by default
+    const bindings = [];
+
     if (this.mode === _CREATE && bindings.length === 0 && this.defaultBindingHandler) {

```

Comment:
what were the scenarios here when in create mode but there were bindings? as we now will always have no bindings